recipes/cobra/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
You can delete this file and add a `script: python setup.py install` line in the build section of the `meta.yaml` file.